netbox/templates/extras/generated_schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "airflow": {
+      "type": "string",
+      "enum": {{ schemas["airflow_choices"] }}
```

Comment:
If it works, would a dot notation be more clearer approah? Example: `{{ schemas.ariflow_choices }}`